app-text/libqxp/libqxp-0.0.1.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="Parses the file format of QuarkXPress"
+DESCRIPTION="libqxp is a library that parses the file format of QuarkXPress \
+documents. Currently it only understands documents created by QuarkXPress \
+3.1 to 4.1."
+HOMEPAGE="https://wiki.documentfoundation.org/DLP/Libraries/libqxp"
+COPYRIGHT="2017-2018 David Tardon, Aleksas Pantechovski"
+LICENSE="MPL v2.0"
+REVISION="1"
+SOURCE_URI="http://dev-www.libreoffice.org/src/libqxp/libqxp-$portVersion.tar.xz"
+CHECKSUM_SHA256="8c257f6184ff94aefa7c9fa1cfae82083d55a49247266905c71c53e013f95c73"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+libVersion=0.0.2
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libqxp$secondaryArchSuffix = $portVersion
+	lib:libqxp$secondaryArchSuffix = $libVersionCompat
```

Comment:
```diff
-	lib:libqxp$secondaryArchSuffix = $libVersionCompat
+	lib:libqxp_0.0$secondaryArchSuffix = $libVersionCompat
```
I found this by looking what haikuporter had left in `app-text/libqxp/work-0.0.1/packaging/libqxp/lib/`.

The `app-text/libqxp/work-0.0.1/packaging/` temporary directory is where haikuporter looks for what it has to compress into each hpkg, which are moved then to `app-text/libqxp/work-0.0.1/hpkgs/`.